src/data/services/StudioApiService.js

```diff
@@ -3,15 +3,27 @@ import { getConfig, ensureConfig } from '@edx/frontend-platform';
 
 ensureConfig([
   'STUDIO_BASE_URL',
+  'LMS_BASE_URL',
 ], 'Studio API service');
 
 const studioBaseUrl = getConfig().STUDIO_BASE_URL;
+const lmsBaseUrl = getConfig().LMS_BASE_URL;
 
 class StudioApiService {
   static getProctoredExamSettingsUrl(courseID) {
     return `${studioBaseUrl}/api/contentstore/v1/proctored_exam_settings/${courseID}`;
   }
 
+  static getCourseDetailsUrl(courseID) {
```

Comment:
I wonder if we should have a separate LmsApiService class for handling interactions with the LMS. I realize they're really the same service due to the monolith, but it feels like it would be beneficial to be explicit about which service we're relying on.

Reply:
I think that's a good idea, I will update the code